src/ad-sizes.ts

```diff
@@ -40,6 +40,10 @@ class AdSize extends Array<number> {
 	get height(): number {
 		return this[1];
 	}
+
+	get array(): number[] {
```

Comment:
What do you think about aligning this with the way we call `toString` above by having a `public toArray()` method rather than a getter?

Reply:
Great shout, that's definitely more consistent! I'll make the change